ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/repositories/LdesFragmentRespository.java

```diff
@@ -1,14 +1,13 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.repositories;
 
-import be.vlaanderen.informatievlaanderen.ldes.server.domain.config.ViewConfig;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesFragment;
 
 import java.util.Optional;
 
 public interface LdesFragmentRespository {
     LdesFragment saveFragment(LdesFragment ldesFragment);
 
-    Optional<LdesFragment> retrieveFragment(String viewShortName, String path, String value);
+    Optional<LdesFragment> retrieveFragmentByViewShortNameAndPathAndType(String viewShortName, String path, String value);
```

Comment:
Not sure if this improves the readability

Reply:
Ha, totally debatable. Usually, I find method names like `retrieveFragment` a bit to vague and like to add some more detail e.g. the argument which is used for selecting the fragment, but I also understand your point of view, ``retrieveFragmentByViewShortNameAndPathAndType` is a pretty long name :-)